packages/security/src/appSecurity/AppSecurity.ts

```diff
@@ -66,4 +64,15 @@ export class AppSecurity {
   public hasConfig(): boolean {
     return !!this.internalConfig;
   }
+
+  /**
+   * Build the init URL from the base URL
+   *
+   * @param baseUrl
+   */
+  private getInitUrl(baseUrl: string): string {
+    const url = new URL(baseUrl);
+    url.pathname = "/api/init";
```

Comment:
is this adding the slash again

Reply:
No. `pathname` replaces the existing forward slash.

https://nodejs.org/api/url.html#url_url_pathname